openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx

```diff
@@ -323,6 +323,12 @@ export const generateEntityLink = (fqn: string, includeColumn = false) => {
   }
 };
 
+export const getEntityFqnFromEntityLink = (entityLink: string) => {
+  const link = entityLink.split('>')[0];
+
+  return link.split('::')[2];
```

Comment:
We should use antlr in the future

Reply:
currently is not working as expected, @ayush-shah  is looking into it in 0.12.1